projects/plugins/jetpack/modules/masterbar/admin-menu/class-atomic-admin-menu.php

```diff
@@ -214,6 +218,28 @@ public function set_site_card_menu_class( array $menu ) {
 		return $menu;
 	}
 
+	/**
+	 * Returns the first available upsell nudge.
+	 *
+	 * @return array
+	 */
+	public function get_upsell_nudge() {
+		$jitm    = \Automattic\Jetpack\JITMS\JITM::get_instance();
+		$message = $jitm->get_messages( 'calypso:sites:sidebar_notice', '{"message_path":"calypso:home:admin_notices"}', false )[0];
```

Comment:
What's that `{"message_path":"calypso:home:admin_notices"}`? Shouldn't be `{"message_path":"calypso:sites:sidebar_notice"}`?